system-addon/lib/TopSitesFeed.jsm

```diff
@@ -137,8 +136,12 @@ this.TopSitesFeed = class TopSitesFeed {
       data: this._getPinnedWithData()
     }));
   }
-  onAction(action) {
+  async onAction(action) {
     switch (action.type) {
+      case at.INIT:
```

Comment:
yeah, I was discussing with @Mardak on IRC: "so adding this refresh means it could refresh earlier than necessary or result in a second refresh after a first NEW_TAB_LOAD"

But we figured that might be better than holding up the refresh when a `NEW_TAB_LOAD` event comes in. Don't you think?